src/ui/prefs.html

```diff
@@ -22,15 +22,21 @@
     <script src="js/extern/jquery-1.9.0.min.js"></script>
     <script>
         $(document).ready(function() {
-          $.getScript('js/Loader.js', function() {
-            Loader.loadScripts([
-                'js/extern/jquery-ui.min.js',
-                'js/extern/spectrum.js',
-                'js/UserPrefs.js'
-            ], function() {
-                Login.showLoginHeader(UserPrefs);
+            $.ajax({
+                url: 'js/Loader.js',
+                dataType: 'script',
+                cache: true,
+                success: function() {
+                    Loader.loadScripts([
+                        'js/extern/jquery-ui.min.js',
+                        'js/extern/spectrum.js',
+                        'js/UserPrefs.js'
+                    ],
```

Comment:
Nit: i actually like this style, where each loaded script is on its own line, better than the style in the other files where the loaded script is inline with Loader.loadScripts.  I think it's easier to read which scripts each file is loading at a glance, which is useful.  Mind changing the others to match this?